### PR TITLE
nb_NO files added.

### PR DESCRIPTION
**Pull request by olekristian**

### CODE REVIEW

File: docs/nb_NO/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+title: General
+lang: en_GB
```
Comment (jxn-30) [inline docs/nb_NO/README.md:3]: Please adjust langcode

File: docs/nb_NO/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+sidebarDepth: 2
+---
+
+# Wiki :gb: <Badge :text="'LSSM V.' + ($themeConfig.variables.versions.stable || 4)"/>
```
Comment (jxn-30) [inline docs/nb_NO/README.md:7]: Please adjust flag

File: docs/nb_NO/other.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+---
+title: Other
+lang: en_GB
```
Comment (jxn-30) [inline docs/nb_NO/other.md:3]: Langcode to nb_NO

File: docs/nb_NO/settings.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+title: Settings ⚙️
+lang: en_GB
```
Comment (jxn-30) [inline docs/nb_NO/settings.md:3]: Also adjust langcode

File: docs/nb_NO/suggestions.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title: Suggestions 💡
+lang: en_GB
```
Comment (jxn-30) [inline docs/nb_NO/suggestions.md:3]: Langcode

File: docs/nb_NO/support.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
+---
+title: Support 👨‍💻
+lang: en_GB
```
Comment (jxn-30) [inline docs/nb_NO/support.md:3]: Langcode

File: src/i18n/nb_NO.ts [hunk 7]
```diff
@@ -0,0 +1,1233 @@
...
+    },
+    globalSettings: {
+        name: 'Innstillinger',
+        labelInMenu: {
+            title: 'Etikett istedenfor ikon i navigasjonslinjen',
+            description:
+                'Viser en enkel etikett i navigasjonslinjen i stedet for LSSM-logoen.',
+        },
+    },
+    vehicles: {
```
Comment (jxn-30) [inline src/i18n/nb_NO.ts:84]: @Suchty112 please check vehicles, buildings and the stuff below.

**Review by jxn-30 — CHANGES_REQUESTED** [review]
See comments.

File: src/i18n/nb_NO.ts [hunk 8]
```diff
@@ -0,0 +1,1233 @@
...
+            caption: 'Ambulanse',
+            color: '#9c1c1c',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+        },
+        6: {
+            caption: 'Røykdykkerbil',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:143]: The next after ambulance is the tankbil pls sort this https://www.nodsentralspillet.com/vehicle_graphics/4
And the other cars further down please

File: src/i18n/nb_NO.ts [hunk 9]
```diff
@@ -0,0 +1,1233 @@
...
+        9: {
+            caption: 'CBRNe enhet',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 11 brannstasjoner.',
+            schooling: 'Krever spesialistutdanning (CBRNe-enhet)',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:181]: Here we need the name as it appears in the course window example: https://prnt.sc/vdzggx

**Review by Suchty112 — CHANGES_REQUESTED** [review]
and the rest is still english. but maybe he wants to change that. so the translation is not finished yet ^

File: src/i18n/nb_NO.ts [hunk 10]
```diff
@@ -0,0 +1,1233 @@
...
+            shownSchooling: 'Hazmat',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:182]: And here is the name as it appears in the personal window  https://prntscr.com/vdzh96

File: src/i18n/nb_NO.ts [hunk 11]
```diff
@@ -369,237 +247,115 @@ export default {
...
+            credits: 19_000,
             minPersonnel: 1,
             maxPersonnel: 2,
-            schooling: 'Police - Sheriff',
-            shownSchooling: 'Sheriff',
+            schooling: 'Politi - Våpentrening',
+            shownSchooling: 'Våpentrening',
         },
     },
     buildings: {
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:323]: the building order is wrong. look here: view-source:https://www.nodsentralspillet.com/buildings/new . after the id's you have to go

File: src/i18n/nb_NO.ts [hunk 12]
```diff
@@ -0,0 +1,873 @@
...
+            color: '#880000',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        19: {
+            caption: 'Lett redningsbåt',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:283]: 19 is Dykkerbil
20 is the boat

File: src/i18n/nb_NO.ts [hunk 13]
```diff
@@ -0,0 +1,873 @@
...
+            coins: 12,
+            credits: 6_000,
+            minPersonnel: 0,
+            maxPersonnel: 0,
+            wtank: 0,
+            special:
+                'Accessory boat that can be towed with the Utility Unit. Requires Swift Water Rescue trained personnel to operate.<br> The boat trailer cannot be assigned personnel, <br>either assign your trained personnel to the towing vehicle, or assign them to a separate unit en route.',
+        },
+        20: {
+            caption: 'Branntankbil',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:294]: Lett redningsbåt

File: src/i18n/nb_NO.ts [hunk 14]
```diff
@@ -0,0 +1,873 @@
...
+            color: '#570f0f',
+            coins: 19,
+            credits: 19_000,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 1500,
+        },
+        21: {
+            caption: 'Utrykningspolitibil',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:303]: Branntankbil

File: src/i18n/nb_NO.ts [hunk 15]
```diff
@@ -0,0 +1,873 @@
...
+            color: '#ad0e0e',
+            coins: 10,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            schooling: 'Politi - Utrykningspoliti',
+            shownSchooling: 'Utrykningspoliti',
+        },
+        22: {
+            caption: 'Utrykningsenhetbil',
```
Comment (Suchty112) [inline src/i18n/nb_NO.ts:313]: Utrykningspolitibil

so the 23 is still missing the Utrykningsenhetbil

File: src/i18n/nb_NO.ts (hunk 11 above)
Comment (Suchty112) [reply src/i18n/nb_NO.ts:323]: That is not resolved. look at the picture the values these are the building id's

**Review by Suchty112 — CHANGES_REQUESTED** [review]

File: src/i18n/nb_NO.ts (hunk 11 above)
Comment (olekristian) [reply src/i18n/nb_NO.ts:323]: I don't understand what you mean here, your example doesn't give me any information other than the list inside ny bygning. https://i.imgur.com/sAf1zJs.png
And that list is corresponding with the list in nb_NO.ts

File: src/i18n/nb_NO.ts (hunk 12 above)
Comment (olekristian) [reply src/i18n/nb_NO.ts:283]: Now corrected, after I discovered the list in https://github.com/LSS-Manager/lss-manager-v3/blob/master/lss-manager-v3/helperfunctions.js

File: src/i18n/nb_NO.ts (hunk 11 above)
Comment (Suchty112) [reply src/i18n/nb_NO.ts:331]: so here all id's
0 = Brannstasjon
1 = Brannskole
2 = Ambulansestasjon
4 = Sykehus
5 = Helikopterstasjon
6 = Politistasjon
7 = Nødetatssenter
8 = Politiskole
13 = Politiheliport
14 = Oppstillingsplass
16 = Fengsel
18 = Brannstasjon (liten)
19 = Politistasjon (liten)
20 = Ambulansestasjon (liten)

Comment (olekristian) [reply src/i18n/nb_NO.ts:331]: Fixed, I feel stupid now - only saw one of your pictures 🤦

File: src/i18n/nb_NO.ts [hunk 16]
```diff
@@ -0,0 +1,861 @@
...
+            caption: 'Delta kjøretøy',
+            color: '#332222',
+            coins: 23,
+            credits: 7_000,
+            minPersonnel: 2,
+            maxPersonnel: 4,
+            special: 'Required once you have built 8 police stations',
+            schooling: 'Police - SWAT',
+            shownSchooling: 'SWAT',
+        16: {
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:236]: Please add a row between row 235 and 236 with **},**

Comment (Suchty112) [reply src/i18n/nb_NO.ts:237]: ahh thx, i dont find the bug :D

File: src/i18n/nb_NO.ts [hunk 17]
```diff
@@ -0,0 +1,862 @@
...
+            }),
+            levelcost: [],
+            maxBuildings: 'No limit',
+            maxLevel: 0,
+            special:
+                "This building can only be built and developed by admins and finance ministers with credits from the association's treasury.The built Prison Cells are available to all members of the association.",
+            startPersonnel: 0,
+            startVehicles: [],
+        },
+    },
```
Comment (jxn-30) [inline src/i18n/nb_NO.ts:630]: Last thing todo before I will merge is sorting these buildings by keys/IDs 😉

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/nb_NO.ts [hunk 18]
```diff
@@ -0,0 +1,863 @@
...
+            caption: 'Brannstasjon',
+            color: '#bb0000',
+            coins: 30,
+            credits: 100_000,
+            extensions: [
+                {
+                    caption: 'Ambulanseutvidelse',
+                    credits: 100_000,
+                    coins: 20,
+                    duration: '7 Days',
```
Comment (Piet2001) [inline src/i18n/nb_NO.ts:323]: days could be translated. I saw it on more buildings.
Also other things in the building information isn't translated.
Sorry, if you are already working on it.

**Review by jxn-30 — APPROVED** [review]

**Comment by olekristian** [comment]
Misclick, @Suchty112

File: src/i18n/nb_NO.ts [hunk 19]
```diff
@@ -0,0 +1,855 @@
...
+        'Liten kjemisk lagringstank',
+        'Stor kjemisk lagringstank',
+        'Hotell',
+        'Bar',
+        'Avfallsdeponi',
+        'Parkeringshus',
+        'Silo',
+        'Lyskryss',
+    ],
+    only_alliance_missions: [41, 61, 62, 112],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:853]: 57 and 74

File: src/i18n/nb_NO.ts [hunk 20]
```diff
@@ -0,0 +1,855 @@
...
+    transfer_missions: [246],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:854]: 77

File: src/i18n/nb_NO.ts [hunk 21]
```diff
@@ -0,0 +1,855 @@
...
+        },
+        7: {
+            caption: 'CBRNe enhet',
+            color: '#770000',
+            coins: 25,
+            credits: 19_200,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            special: 'Nødvendig etter at du har bygd 11 brannstasjoner.',
+            schooling: 'Krever spesialistutdanning (CBRNe-enhet)',
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:155]: Is this one exactly the same as in the course window? I think the category and the **-** are missing.

File: src/i18n/nb_NO.ts [hunk 22]
```diff
@@ -0,0 +1,855 @@
...
+            color: '#00ac00',
+        },
+    },
+    small_buildings: {
+        0: 13,
+        3: 16,
+        5: 15,
+    },
+    vehicleBuildings: [0, 3, 5, 6, 8, 9, 11, 12, 13, 14, 15, 16, 17, 18],
+    cellBuildings: [5],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:687]: 6

File: src/i18n/nb_NO.ts [hunk 23]
```diff
@@ -0,0 +1,855 @@
...
+    cellExtensions: [
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:688]: '6_0'
'6_1' and up

File: src/i18n/nb_NO.ts [hunk 24]
```diff
@@ -0,0 +1,855 @@
...
+                buildingsAmountTotal < 125
+                    ? 4
+                    : Math.floor(buildingsAmountTotal / 25),
+        },
+        6: {
+            caption: 'Politistasjon',
+            color: '#007700',
+            coins: 35,
+            credits: 100_000,
+            extensions: [
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:471]: Does the first cell have a different name from the rest?

File: src/i18n/nb_NO.ts [hunk 25]
```diff
@@ -0,0 +1,855 @@
...
+        'Politi': {
+            buildings: [7, 8, 10, 13, 9],
+            color: '#00ac00',
+        },
+        'Annet': {
+            buildings: [0, 12],
+            color: '#02a18c',
+        },
+    },
+    vehicleCategories: {
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:650]: Please translate the categorynames and groups

File: src/i18n/nb_NO.ts [hunk 26]
```diff
@@ -0,0 +1,855 @@
...
+                'K-9 Unit': [13],
+                'Sheriff': [21],
+            },
+            color: '#00ac00',
+        },
+    },
+    small_buildings: {
+        0: 13,
+        3: 16,
+        5: 15,
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:684]: 6: 19

File: src/i18n/nb_NO.ts [hunk 27]
```diff
@@ -0,0 +1,855 @@
...
+                'Police helicopter': [11],
+                'K-9 Unit': [13],
+                'Sheriff': [21],
+            },
+            color: '#00ac00',
+        },
+    },
+    small_buildings: {
+        0: 13,
+        3: 16,
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:683]: 2: 20

File: src/i18n/nb_NO.ts [hunk 28]
```diff
@@ -0,0 +1,855 @@
...
+                'Police Motorcycle': [14],
+                'Police helicopter': [11],
+                'K-9 Unit': [13],
+                'Sheriff': [21],
+            },
+            color: '#00ac00',
+        },
+    },
+    small_buildings: {
+        0: 13,
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:682]: 0: 18

File: src/i18n/nb_NO.ts [hunk 29]
```diff
@@ -0,0 +1,855 @@
...
+        '5_3',
+        '5_4',
+        '5_5',
+        '5_6',
+        '5_7',
+        '5_8',
+        '5_9',
+    ],
+    bedBuildings: [2, 14],
+    schoolBuildings: [4, 7],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:701]: 1, 8

File: src/i18n/nb_NO.ts [hunk 30]
```diff
@@ -0,0 +1,855 @@
...
+        '5_2',
+        '5_3',
+        '5_4',
+        '5_5',
+        '5_6',
+        '5_7',
+        '5_8',
+        '5_9',
+    ],
+    bedBuildings: [2, 14],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:700]: 4

File: src/i18n/nb_NO.ts [hunk 31]
```diff
@@ -0,0 +1,855 @@
...
+    schoolBuildings: [4, 7],
+    dispatchCenterBuildings: [0],
```
Comment (JRH-1997) [inline src/i18n/nb_NO.ts:702]: 7

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 32]
```diff
@@ -0,0 +1,93 @@
...
+	},
+	"prisoners": {
+		"max_possible_prisoners": "Maksimalt antall fanger",
+		"title": "Fange | {n} fange | {n} fanger"
+	},
+	"requirements": {
+		"water_needed": "Water demand (gal)"
+	},
+	"vehicles": {
+		"captions": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.json:41]: afbeelding
Only need the above categories

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 33]
```diff
@@ -0,0 +1,93 @@
...
+		"patient_allow_first_responder_chance": "Sannsynlighet for First Responder:",
+		"patient_at_end_of_mission": "NB: Pasienten dukker opp på slutten av oppdraget!",
+		"patient_transport": "Sannsynlighet for transport:",
+		"possible_patient": "Maksimalt antall pasienter",
+		"possible_patient_exact": "kun {n} pasient | kun {n} pasienter",
+		"possible_patient_min": "minimum pasienter",
+		"specializations": "Utdanning/Spesialitet",
+		"title": "Pasienter | {n} pasient | {n} Pasienter"
+	},
+	"prerequisites": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.json:21]: afbeelding
Needs only the above catgories

File: src/modules/missionHelper/i18n/nb_NO.json (hunk 32 above)
Comment (JRH-1997) [reply src/modules/missionHelper/i18n/nb_NO.json:41]: And add **trafic_car_only_if_needed** for the only if present option. Maybe Sanni or Jan could make the settings for that when  you don't know how to do in the settingfiles and root file.

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 34]
```diff
@@ -0,0 +1,93 @@
...
+				"additional_text": "",
+				"reduce_from": "mobile_command_vehicles"
+			},
+			"heavy_rescue_vehicles": {
+				"additional_text": "thereof at least. {n} Rescue Engines",
+				"reduce_from": "firetrucks"
+			}
+		},
+		"optional_alternatives": {
+			"allow_arff_instead_of_lf": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.json:81]: not in nb_NO

File: src/modules/missionHelper/i18n/nb_NO.json [hunk 35]
```diff
@@ -0,0 +1,93 @@
...
+				"firetrucks": "Fire Engine or ARFF | Fire Engine or ARFF"
+			},
+			"allow_rw_instead_of_lf": {
+				"firetrucks": "Fire Engine or Heavy Rescue or Rescue Engine | Fire Engines or Heavy Rescue or Rescue Engine"
+			}
+		},
+		"patient_additionals": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.json:88]: Do you have any requirements when you have more than {number} patients?

File: src/modules/missionHelper/i18n/nb_NO.json (hunk 34 above)
Comment (JRH-1997) [reply src/modules/missionHelper/i18n/nb_NO.json:81]: Please translate the rw instread of lf. It's: "Tungredningsbil or  Mannskapsbil"

File: src/modules/missionHelper/i18n/nb_NO.root.json [hunk 36]
```diff
@@ -0,0 +1,151 @@
...
+				"description": "Trekker antall nødvendige brannbiler fra tungredningsbiler og skjuler brannbiler hvis tallet faller under 0. Denne funksjonen er ment for personer som kjører et rent redningsbil-system.",
+				"title": "Tungredningsbil vs redningsbil"
+			},
+			"platform_trucks": {
+				"description": "Trekker antall brannbiler fra Snorkelbiler og skjuler brannbiler hvis tallet faller under 0.",
+				"title": "Snorkelbil vs Stigebil"
+			}
+		},
+		"optionalAlternatives": {
+			"allow_arff_instead_of_lf": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.root.json:58]: This one isn't needed

File: src/modules/missionHelper/i18n/nb_NO.root.json [hunk 37]
```diff
@@ -0,0 +1,151 @@
...
+		},
+		"hide_battalion_chief_vehicles": {
+			"description": "Skjuler innsatsleder brann og viser heller innsatsstøttebil.",
+			"title": "Skjul innsatsleder brann"
+		},
+		"id": {
+			"description": "Viser oppdrags ID etter oppdragsnavnet.",
+			"title": "Oppdrags ID"
+		},
+		"k9_only_if_needed": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.root.json:39]: This one isn't needed, but you can add here the one for traffic_car

File: src/modules/missionHelper/i18n/nb_NO.root.json [hunk 38]
```diff
@@ -0,0 +1,151 @@
...
+		"optionalAlternatives": {
+			"allow_arff_instead_of_lf": {
+				"description": " Indicates, if instead of a firetruck arrfs are possible on site.",
+				"title": "Show if arrfs are allowed as firetrucks"
+			},
+			"allow_ktw_instead_of_rtw": {
+				"description": "Viser om en syketransport kan benyttes istedenfor en ambulanse.",
+				"title": "Syketransport istedenfor ambulanse"
+			},
+			"allow_rw_instead_of_lf": {
```
Comment (JRH-1997) [inline src/modules/missionHelper/i18n/nb_NO.root.json:66]: Please translate its the setting if Tungredningsbil  could be a Mannskapsbil

Comment (JRH-1997) [reply src/modules/missionHelper/i18n/nb_NO.root.json:66]: And please translate the rest of this file, so you have done this one.

File: src/modules/extendedCallWindow/i18n/nb_NO.js [hunk 39]
```diff
@@ -0,0 +1,169 @@
...
+    arrHover: {
+        reset: 'The vehicle selection is reset before!',
+        titles: {
+            set: 'So often this combination is selected when you press the AAR',
+            attribute: 'The combination described in this line',
+            free: 'So often this combination is available',
+            max:
+                'You can select this AAR until this combination is no longer available',
+        },
+        arrSpecs: {
```
Comment (JRH-1997) [inline src/modules/extendedCallWindow/i18n/nb_NO.js:24]: You can add this now and do the rest at a later moment:
        fire: 'Brannbil', 
        elw: 'Innsatsleder brann', 
        dlk: 'Stigebil', 
        rtw: 'Ambulanse', 
        rw: 'Tungredningsbil', 
        elw2: 'Innsatsstøttebil', 
        elw1_or_elw2: 'Innsatsleder brann eller innsatsstøttebil', 
        hlf_only: 'Redningsbil', 
        hlf_or_rw_and_lf: 'Redningsbil eller Tungredningsbil og Brannbil', 
        gwa: 'Røykdykkerbil', 
        gwl2wasser: 'Tankbil', 
        gwgefahrgut: 'CBRNe-enhet', 
        fustw: 'Patruljebil', 
        police_motorcycle: 'Politimotorsykkel', 
        fustw_or_police_motorcycle: 'Patruljebil eller Politimotorsykkel', 
        polizeihubschrauber: 'Politihelikopter', 
        swat: 'Politi med våpentrening', 
        swat_armored_vehicle: 'Pansret kjøretøy', 
        swat_suv: 'Delta kjøretøy', 
        any_traffic_car: 'Utrykningspolitibil', 
        k9: 'Hundepatrulje', 
        rth_only: 'Luftambulanse', 
        gw_taucher: 'Dykkerbil', 
        boot: 'Båter (generelt)', 
        mzb: 'Lett redningsbåt',

**Review by JRH-1997 — CHANGES_REQUESTED** [review]
I've made some comments. Please look at that. 
You could translate the rest later, but not all functions are working with missing translations or ID's and stuff like that.

File: src/modules/extendedCallWindow/i18n/nb_NO.js [hunk 40]
```diff
@@ -0,0 +1,169 @@
...
+            [/^Wildland fire engine(s)$/]: [30, 31, 32, 38, 39, 40],
+            [/^Fire Aviation(s)$/]: [35, 36, 37],
+            [/^FBI Unit(s)$/]: [42],
+            [/^FBI Investigation Wagon(s)$/]: [43],
+            [/^FBI Mobile Command Center(s)$/]: [44],
+            [/^FBI Bomb Technician Vehicle(s)$/]: [45],
+            [/^FBI Surveillance Drone(s)$/]: [46],
+            [/^Sheriff Unit(s)$/]: [47],
+        },
+    },
```
Comment (Piet2001) [inline src/modules/extendedCallWindow/i18n/nb_NO.js:110]: This are not the names that will be shown in the red bar.
So I think this wil result in an error for the table.

File: src/modules/extendedCallWindow/i18n/nb_NO.js [hunk 41]
```diff
@@ -0,0 +1,169 @@
...
+    tailoredTabs: {
+        defaultTabs: [
```
Comment (Piet2001) [inline src/modules/extendedCallWindow/i18n/nb_NO.js:112]: this are also not the defaultTabs of the game I think

**Review by Piet2001 — CHANGES_REQUESTED** [review]

File: src/modules/extendedCallWindow/i18n/nb_NO.js (hunk 40 above)
Comment (jxn-30) [reply src/modules/extendedCallWindow/i18n/nb_NO.js:110]: no error, but table will not work. Can be done later actually. It is not easy to translate without having a basic translation on live system ;)

**Review by Suchty112 — APPROVED** [review]
Is approved first. the translations can be added later

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**